templates/anndata_conversion.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<p>This should go in the <code>development.md</code> section. You can see an example of how we document <code>xarray</code>. On a side note, feel free to open a PR to add onto the existing <code>xarray</code> documentation there.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1079/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>